### PR TITLE
Create original files node

### PR DESCRIPTION
**Pull request by AlbertSnows**
Link to JIRA ticket if there is one: https://ctds-planx.atlassian.net/jira/software/c/projects/BDC/boards/153?selectedIssue=BDC-215

### New Features
- Adding a new node to the BDC graph for original files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -80,3 +80,10 @@ admin_auth
 
 # artifacts
 artifacts/
+/.idea/.gitignore
```
Comment (k-burt-uch) [inline .gitignore:83]: all you need to add is ".idea"

**Review by k-burt-uch — DISMISSED** [review]

File: gdcdictionary/schemas/original_file.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "original_file"
+title: Original File
+type: object
+namespace: https://dcp.bionimbus.org/
+category: original_data
+program: '*'
+project: '*'
+description: "A storage node for the original, raw contents of a study ingested into our system."
```
Comment (k-burt-uch) [inline gdcdictionary/schemas/original_file.yaml:10]:
```suggestion
description: "A storage node for the original data files of a study. This node is only populated for dataset level PFBs added to Discovery."
```

Comment (k-burt-uch) [reply gdcdictionary/schemas/original_file.yaml:10]: I think we need to be clear that this node will not have data in the graph. @Avantol13 @jdorsheimer any suggestions on wordsmithing?

**Review by k-burt-uch — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/original_file.yaml (hunk 2 above)
Comment (Avantol13) [reply gdcdictionary/schemas/original_file.yaml:10]: eh, I don't know if that's absolutely necessary to be in this description. I think it's fine if that's a particular pattern/rule we follow without encoding it here.

Comment (k-burt-uch) [reply gdcdictionary/schemas/original_file.yaml:10]: Sounds good. @AlbertSnows ignore my suggestion.

**Review by k-burt-uch — APPROVED** [review]

**Pull request merged**